### DISCUSSION TITLE
HRR curve, Chemical reactions and CO/Soot yield parameters

### DISCUSSION THREAD

**Discussion by skranus** [firemodels/fds#12940] [Q&A]
Hi everyone, for a project I need to define HRR curves, chemical combustion reactions with related CO/soot parameters of PP, PS and PE (Polypropylene, Polystyrene and Polyethylene).

I have already found the chemical formulae and corresponding (balanced) combustion reaction of each of the three materials, I know the melting point of all of them, the heat of combustion (KJ/Kg) and the heat with the oxygen consumption method (KJ/g[O2]).

I would just need to be able to find the HRR curves of the three materials, but I couldn't find much on the web, not even the SFPE (5th edition)

Could someone help me?
thanks in advance

**Comment by drjfloyd** [comment]
The HRR curve is going to be a function of the quantity of material, the geometric arrangment of the material, the location and manner of igniton, etc. There is no single HRR curve.  Try searching for test data on objects containing those parameters, cone test data, etc.  You may wish to look at the User's Guide section on scaling pyrolysis and the associated cases in the Validation Guide which reference some online resources.

**Comment by TristanHehnen** [comment]
In the [FSRI Materials Database](https://github.com/ulfsri/fsri_materials_database) is some information for [PP](https://github.com/ulfsri/fsri_materials_database/tree/main/01_Data/PP). 

Maybe that helps.
